bridges/AmazonPriceTrackerBridge.php

```diff
@@ -146,13 +146,13 @@ private function scrapePriceFromMetrics($html) {
 	}
 
 	private function scrapePriceGeneric($html) {
-		$priceDiv = $html->find('span.offer-price', 0) ?: $html->find('.a-color-price', 0);
+		$priceDiv = $html->find('#priceblock_ourprice', 0) ?: $html->find('.priceBlockBuyingPriceString', 0);
```

Comment:
Think this will break in other countries and cases where this div isn't set. 

Adding these changes to #2205 for a exhaustive approach.

Reply:
Tested in in US, Italy and Germany, with subscription products, regular products and also discounted products.

But yeah.. it's always prone to fail, as there are so many different display styles on the various Amazon product pages.